README.md

```diff
@@ -1,387 +1,368 @@
+
+### Slot Manipulation
+
+We need this one to create an alarm component out of a blockquote with an icon
+in its first line like below;
+
+> :warning:
+>
+> Demo warning message
+
+Content of this blockquote is passed to `ProseBlockquote` component in
+`<slot/>`. To parse the first line of this slot, we used `useSlots` in script
+where nuxt gives access to element tree of a markdown content.
+
+`<slot/>` does not allow to manipulate its content, so we used `<component
+:is="..."/>` instead. This is the way to render a slot that is programmatically
+changed or created.
+
+Other types demonstrated below;
+
+> :information_source:
+>
+> Demo info message
+
+> Demo default message
+
+### Disabling Emoji Conversion
+
+Nuxt content comes with a default setting that converts emoji texts into emoji
+icons. This is disabled in `.theme/nuxt.config.ts` under
+`content.markdown.remarkPlugins.remark-emoji`. This way a prose component gets
+original text instead of an emoji icon which is better because we don't want to
+place an emoji icon in code.
+
+## Provide & Inject
+
+When you need to access a data that cannot be passed to a component (such as
+prose components), we use `provide` and `inject` functions. Basically you
+`provide` a value in a parent component to be `inject`ed in any child
+components.
+
+```mermaid
+flowchart TB
+    subgraph C[Components]
+        direction TB
+
+        C_P[Page]
+        C_CD[&ltContentDoc/&gt]
+        C_PA[ProseA]
+        C_P -- renders --> C_CD -- uses --> C_PA
+    end
+
+    D([data])
+
+    C_P -. provides .-> D -. injected to .-> C_PA
+```
+
+Demo is at [/demo/provide-and-inject](/demo/provide-and-inject)
+
+## Escape Curly Braces
+
+To escape curly braces use `v-pre` as shown in
+`.theme/pages/demo/escape-curly-braces.vue`.
+
+Demo is at [/demo/escape-curly-braces](/demo/escape-curly-braces).
+
+## Open Graph
+
+With Open Graph, you can offer a preview of your site with given title,
+description, image and url for the preview in your meta.
+
+We can change meta in 2 ways in nuxt.
+
+### In `nuxt.config.ts`
+
+You can give the head of your page by default. You can find an example
+in `.theme/nuxt.config.ts`.
+
+To see this metadata, view source of this page.
+
+### Using `useHead()`
+
+To update the metas with `useHead()` you can access the head content from
+your `.vue` files.
+
+Demo is at [/demo/open-graph](/demo/open-graph)
+
+To see the changes, go to [/demo/open-graph](/demo/open-graph) and view the
+source code
+
+## Build Process
+
+The build process proceeds in 3 stages. In the first stage, the preparation we
+made for markdowns before the prebuild is working. You can review
+[content / preprocessing-mardown-files](/content/preprocessing-mardown-files)
+for this stage. The second stage includes builds. The last stage is divided
+into two. In the last stage, if we want to deploy, we create a static site
+using `nuxt generate`. If we want to stand up in developer mode, the vite
```

Comment:
turkcede ayaga kalkmak diyoruz da, ingilizcede stand up olmaz ya. `If we want to run it in development mode` denebilir